Sources/AppCoinsSDK/Models/Services/Transaction/AppCoinTransactionClient.swift

```diff
@@ -31,14 +31,21 @@ internal class AppCoinTransactionClient : AppCoinTransactionService {
                 let task = URLSession.shared.dataTask(with: request) { data, response, error in
                     if let error = error {
                         if let nsError = error as NSError?, nsError.code == NSURLErrorNotConnectedToInternet {
-                            result(.failure(.noInternet))
+                            result(.failure(.noInternet(message: "Internet Connection Failed", description: "Could not get internet connection to \(url)", request: DebugRequestInfo(request: request, responseData: data, response: response))))
```

Comment:
Also add code location on the description or service layer errors. Do it for all errors on this file, I will not mark them all.

Reply:
Done